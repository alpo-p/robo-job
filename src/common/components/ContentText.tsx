import React from 'react'
import RoboText from './RoboText'

const ContentText: React.FC = ({ children }) => (
  <RoboText
    weight="light"
    size="small"
    style={{
      lineHeight: 24,
      textAlign: 'justify',
    }}
  >
    {children}
  </RoboText>
)

export default ContentText
