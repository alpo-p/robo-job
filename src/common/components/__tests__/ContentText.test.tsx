import React from 'react'
import renderer from 'react-test-renderer'
import ContentText from '../ContentText'

describe('CompanyLogo', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<ContentText>Testi</ContentText>)
    expect(tree).toMatchSnapshot()
  })
})
