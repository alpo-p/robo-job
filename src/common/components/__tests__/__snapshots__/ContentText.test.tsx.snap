// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompanyLogo should match snapshot 1`] = `
<Text
  style={
    Object {
      "color": "rgb(28, 28, 30)",
      "fontFamily": "DamascusLight",
      "fontSize": 14,
      "lineHeight": 24,
      "textAlign": "justify",
      "textTransform": "none",
    }
  }
>
  Testi
</Text>
`;
